integtest/spec/all_books_spec.rb

```diff
@@ -55,8 +55,10 @@ def self.has_license(name, heading)
     end
     file_context 'html/static/styles.css' do
       has_license 'Inter', 'SIL OPEN FONT LICENSE'
+      has_license 'Noto Sans Japanese', 'SIL OPEN FONT LICENSE'
     end
     file_context 'html/static/Inter-Medium.5d08e0ba.woff2'
+    file_context 'html/static/NotoSansJP-Black.woff2'
```

Comment:
I wasn't certain what the purpose of running `file_context` without a block was, so I just added this to, presumably, verify that some Noto Sans file is present at build time.

Reply:
It just checks if the file exists.